web/bind.go

```diff
@@ -77,6 +81,41 @@ func dumpError(err error, logger echo.Logger, dump []byte) {
 	}
 }
 
+func (b *DefaultBinder) bindDefaultData(ptr interface{}) error {
+	typ := reflect.TypeOf(ptr).Elem()
+	val := reflect.ValueOf(ptr).Elem()
+
+	if typ.Kind() != reflect.Struct {
+		return NewHTTPError(http.StatusBadRequest, "binding element must be a struct")
+	}
+
+	for i := 0; i < typ.NumField(); i++ {
+		typeField := typ.Field(i)
+		structField := val.Field(i)
+		if !structField.CanSet() {
+			continue
+		}
+
+		defaultValue := typeField.Tag.Get("default")
+		if len(defaultValue) == 0 {
```

Comment:
if we have something that has come, but by default it's not empty-it will overwrite by default? or it seems to me
т. е. если у нас что то пришло, но default не пустой- он перезапишет на default? или мне кажется

Reply:
кажется, сначала устанавливаются все значения по дефолту, потом устанавливаются в зависимости что пришло